### ISSUE TITLE
Can there be a lint warning for productions not referenced by emu-prodref?

### ISSUE THREAD

**Issue by gibson042** [tc39/ecmarkup#485]
cf. https://github.com/tc39/ecma262/pull/2912

It would ideally be configurable and/or suppressible, because ECMA-262 maintains a grammar summary but I don't think _every_ spec does.

**Comment by michaelficarra** [comment]
I'd rather just figure out a way to auto-generate Annex A.

**Comment by gibson042** [comment]
I agree, but that may be significantly more difficult (at least with its current organization).
